bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/AbstractKNXClient.java

```diff
@@ -299,6 +300,11 @@ private void readNextQueuedDatapoint() {
                     logger.warn("Giving up reading datapoint {}, the number of maximum retries ({}) is reached.",
                             datapoint.getDatapoint().getMainAddress(), datapoint.getLimit());
                 }
+            } catch (KnxRuntimeException e) {
+                // KnxRuntimeException is _not_ a subclass of KnxException. Fail gracefully for this case as well,
+                // fixes #7239
```

Comment:
I don't think this is necessary to mention the github issue number directly in code. Please remove it.

Reply:
will do